plugins/modules/database/saphana/hana_query.py

```diff
@@ -0,0 +1,191 @@
+        type: str
+        required: true
+    user:
+        description: A dedicated username. Defaults to C(SYSTEM).
+        type: str
+        default: SYSTEM
+    password:
+        description: The password to connect to the database.
+        type: str
+        required: true
+    autocommit:
+        description: Autocommit the statement.
+        type: bool
+        default: True
+    host:
+        description: The Host IP address. The port can be defined as well.
+        type: str
+    database:
+        description: Define the database on which to connect.
+        type: str
+    encrypted:
+        description: Use encrypted connection. Defaults to C(false).
+        type: bool
+        default: false
+    filepath:
+        description:
+        - One or more files each containing one SQL query to run.
+        - Must be a string or list containing strings.
+        type: list
+        elements: path
+    query:
+        description:
+        - SQL query to run. Multiple queries can be passed using YAML list syntax.
+        - Must be a string or list containing strings.
+        type: list
+        elements: str
+
+author:
+    - Rainer Leber (@rainerleber)
+'''
+
+EXAMPLES = r'''
+
+- name: Simple select query
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query: "select user_name from users"
+
+- name: Run several queries
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query:
+    - "select user_name from users;"
+    - select * from SYSTEM;
+    host: "localhost"
+    autocommit: False
+
+- name: Run several queries from file
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    filepath:
+    - /tmp/HANA_CPU_UtilizationPerCore_2.00.020+.txt
+    - /tmp/HANA.txt
+    host: "localhost"
+
+
+'''
+
+import io
+import csv
+import json
+from ansible.module_utils.basic import AnsibleModule
+
+
+def csv_to_json(rawcsv):
+    lines = rawcsv[:rawcsv.rfind('\n')]
+    reader_raw = csv.DictReader(io.StringIO(lines))
+    reader = [dict((k, v.strip()) for k, v in row.items()) for row in reader_raw]
+    tolist = list(reader)
+    return json.dumps(tolist)
+
+
+def main():
+    global module
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            sid=dict(type='str', required=True),
+            instance=dict(type='str', required=True),
+            encrypted=dict(type='bool', required=False, default=False),
+            host=dict(type='str', required=False),
+            user=dict(type='str', required=False, default="SYSTEM"),
+            password=dict(type='str', required=True, no_log=True),
+            database=dict(type='str', required=False),
+            query=dict(type='list', elements='str', required=False),
+            filepath=dict(type='list', elements='path', required=False),
+            autocommit=dict(type='bool', required=False, default=True)
```

Comment:
```suggestion
            autocommit=dict(type='bool', required=False, default=True),
```
That way the diff will be smaller when adding another parameter in the future.